src/test/java/com/bootexample4/RoostTest/create-profile-accountPostTest.java

```diff
@@ -0,0 +1,198 @@
+// Test generated by RoostGPT for test test-postman1 using AI Type Open AI and AI Model gpt-4-1106-preview
+
+
+// Test generated for /order-service/orders/create-profile-account_post for http method type POST in rest-assured framework
+
+
+// RoostTestHash=7b184cbf3a
+
+
+package com.bootexample4.RoostTest;
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import io.restassured.response.Response;
+import org.junit.jupiter.api.Test;
+import static io.restassured.RestAssured.given;
+import static org.junit.Assert.*;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.File;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.List;
+import org.hamcrest.MatcherAssert;
+import static org.hamcrest.Matchers.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class create-profile-accountPostTest {
+  
+    @Test  
+    public void create-profile-account_post_Test() {  
```

Comment:
method name should not contain - , it will throw syntax error. Instead use `create_profile_account_post_Test`